src/pages/console/uibuilder/slots.mdx

```diff
@@ -0,0 +1,150 @@
+1. On the left-hand side, you'll see the Elements of your component.  Select a Figma Frame (![Figma frame icon](/images/console/figma-frame-icon.png)) within your component.  
+
+  In this example, the "Area" frame is selected.
+
+2. On the right-hand bar action bar, click the "Convert to a slot" button.  This will add a new component under the "Component properties" header.
+
+3. Optionally, change your your property name.  
+
+  In this example, the property has been renamed "comments"
+
+![Select your frame and add a slot](/images/console/slots-console2.gif)
+
+<Callout warning>
+
+Want to undo your slot creation?  
+
+Locate your slot in the Component properties, click the triple-dot icon, and click "Erase property" to remove the slot.
+
+</Callout>
+
+## Importing your component
+Once you've added a slot to your component, click the "Get component code" button at the bottom of the screen to see instructions on the next steps.  
+1. Copy the `amplify pull` command, and run it in your Terminal
+2. Copy the import code and paste it in your App.js file
+3. Lastly, render the component by calling it in JSX
+
+```jsx
+/*Import your component*/
+import { 
+  Ampligram 
+} from './ui-components'
+
+function App() {
+  return (
+    /*Add your component below*/
+    <div style={styles.post}>
+      <Ampligram  /> 
+    </div>
+  );
+}
+
+const styles = {
+  post: { width: 400, margin: '0 auto', padding: 20},
+}
+
+export default App;
+
+```
+
+Here's how the code above would render in your app.  Some minor styling has been added to help with visibility.
+
+![Imported component with a slot](/images/console/slots-base.png)
+
+## Using your slot
+To use the slot, pass a child component as a property of the parent component, using the prop name you picked earlier.  Then, the content you pass will be rendered as a child of the component.  
+
+
+````jsx
+import { 
+  Ampligram 
+} from '.ui-components'
+
+function App() {
+  return (
+    <div style={styles.post}>
+      <h1>Simple Slot Children</h1>
+      <Ampligram comments={
+        /* Pass your content into the slot */
+        <div style={styles.comment}> 
+          <h1>Hi mom!</h1>
+          <p>Thanks for checking out my app</p>
+        </div>
+      }/>
+    </div>
+  );
+}
+
+const styles = {
+  post: { width: 400, margin: '0 auto', padding: 20},
+  comment: { paddingLeft: 30, textAlign: 'left'},
+}
+
+export default App;
+
+````
+
+Here's how the code above would render in your app.  We've added a little more styling to the comments to help with display.
+
+![Simple slot children](/images/console/slots-simple-children.png)
+
+## Adding slots to collections
+Any component can be [converted to a collection](/console/uibuilder/collections/) and bound to data, and that includes components with slots.  With a collection, you can [extend your component via code](/console/uibuilder/override/) using the `overrideItems` prop to generate unique content within each component slot.  
```

Comment:
Please do a full pass on all content and remove "double spacing" after period. Just like to keep it consistent across the docs with `. More content.` instead of `.  More content.`